tests/test_specials.py

```diff
@@ -101,3 +109,56 @@ def test_slash_h_alias(executor):
     h_results = executor('\h SELECT')
     results = executor('\? SELECT')
     assert results[3] == h_results[3]
+
+
+@contextmanager
+def temp_filepath(filename):
+    try:
```

Comment:
Thanks!  I should be able to incorporate that later this week.

On Mon, Jun 6, 2016 at 11:05 PM, Daniel Rocco <notifications@github.com>
wrote:

> In tests/test_specials.py
> <https://github.com/dbcli/pgspecial/pull/16#discussion_r66003859>:
>
> > @@ -101,3 +109,56 @@ def test_slash_h_alias(executor):
> >      h_results = executor('\h SELECT')
> >      results = executor('\? SELECT')
> >      assert results[3] == h_results[3]
> > +
> > +
> > +@contextmanager
> > +def temp_filepath(filename):
> > +    try:
>
> Hey @catherinedevlin <https://github.com/catherinedevlin>, py.test
> provides a fixture that does this already:
> http://pytest.org/latest/tmpdir.html
>
> :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dbcli/pgspecial/pull/16/files/e3d31fbd384dcc9a7e6bc65ba6a1c63452d1cde7#r66003859>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAMstv3-DN2Z3z4DxFVGciFl70u-DtJ3ks5qJOALgaJpZM4ItNaA>
> .
>



-- 
- Catherine
http://catherinedevlin.blogspot.com